test/tools/integration/Makefile

```diff
@@ -1,3 +1,6 @@
+# prow uses BUILD_ID
+# TODO: remove CIRCLE_BUILD_NUM entirely after moving to prow
+CIRCLE_BUILD_NUM=$BUILD_ID
```

Comment:
This needs a second `$` sign to be `$$BUILD_ID`, otherwise the value of `CIRCLE_BUILD_NUM` will be `UILD_ID`

Reply:
Done.